src/entries/DailyEntry.js

```diff
@@ -49,59 +48,33 @@ export class DailyEntry extends Component {
     this.setState({ inputs: result });
   }
 
-
   render() {
     return (
-      <View style={styles.containerStyle}>
-
-        <TextInput
-          ref={(component) => { this.inputToClear = component; }}
-          style={styles.textInputStyle}
-          onChangeText={text => this.setState({ input: text })}
-          multiline
-          placeholder={this.props.placeholder}
-        />
-
-        {this.state.inputs.map((element, index) =>
+      <Container>
+        <Item regular>
+          <Input
+            ref={(component) => { this.inputToClear = component; }}
+            onChangeText={text => this.setState({ input: text })}
+            placeholder={this.props.placeholder}
+            multiline
+            numberOfLines={3}
```

Comment:
i think. It should be higher than 3 lines. But i will have to see it in real screen